### PR TITLE
Convert "CountOperationTest" unit test to use native CQL driver

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

Fixes one currently disabled unit test ("CountOperationTest")

**Which issue(s) this PR fixes**:
N/A (filed on wrong issue repository)

**Checklist**
- [ ] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by tatu-at-datastax** [comment]
@maheshrajamani Sending this for review to validate that this approach makes sense, before converting the rest.

File: src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/OperationTestBase.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class OperationTestBase {
+  protected final String KEYSPACE_NAME = RandomStringUtils.randomAlphanumeric(16);
+  protected final String COLLECTION_NAME = RandomStringUtils.randomAlphanumeric(16);
+  protected final CommandContext CONTEXT = new CommandContext(KEYSPACE_NAME, COLLECTION_NAME);
+
+  protected ColumnDefinitions buildColumnDefs(List<String> columnNames, List<Integer> typeCodes) {
```
Comment (maheshrajamani) [inline src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/OperationTestBase.java:23]: Can we use List<Pair<>> instead of 2 array

**Review by maheshrajamani — APPROVED** [review]
LGTM.

File: src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/OperationTestBase.java (hunk 1 above)
Comment (tatu-at-datastax) [reply src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/OperationTestBase.java:23]: Sure, let me see which Pair we have available.

Comment (tatu-at-datastax) [reply src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/OperationTestBase.java]: Commons-lang3 has Pair, but also realized that it's even cleaner to just add and use `record` for information.

**Pull request merged**